docs/Upgrade.rst

```diff
@@ -457,18 +197,19 @@ Prerequisites
 Before the upgrade, you need to:
 
 -  Back up data, as described in :ref:`Backup and Restore <Backup>`, 
-   before upgrading. In case anything turns badly, your data is safe and *
+   before upgrading. In case anything turns badly, your data is safe and 
    you can start over.
 
 -  Back up customizations (including configuration, deployed extensions
    and applications) that you plan to reuse in the new version.
 
 -  Download eXo Platform 5.1 version.
 
--  eXo Platform 5.1 requires :ref:`Elasticsearch 5.6 <Elasticsearch.ES_External>`
-   , so you must upgrade your Elasticsearch cluster to this version. 
-   Please refer to the `Elasticsearch documentation <https://www.elastic.co/guide/en/elasticsearch/reference/5.6/setup-upgrade.html>`__
-   to perform this upgrade.
+-  The configuration properties for caches have been changed for eXo 
+   Platform 5.1 in order to harmonize them and use the same name in 
+   standalone and cluster modes. You should then update cache properties
+   in the new server. Please refer to this :ref:`documentation <Configuration.CacheConfiguration>`
```

Comment:
I would change 
"You should then update cache properties in the new server. Please refer to this for get the list of cache names." 
to
"In case you changed default cache configuration, take care to update the cache properties names as documented at <lilnk>."
WDYT ?